test/integration/verify/controls/aws_guardduty_detectors.rb

```diff
@@ -0,0 +1,14 @@
+title 'Test AWS GuardDuty Detectors in bulk'
+
+aws_guardduty_detector_id = attribute(:aws_guardduty_detector_id, default: '', description: 'The AWS GuardDuty Detector ID.')
+
+control 'aws-guardduty-detectors-1.0' do
+
+  impact 1.0
+  title 'Ensure AWS GuardDuty Detector has current properties'
+
+  describe aws_guardduty_detectors do
```

Comment:
It would be good to show an example of this being used in conjunction with the singular resource. Refer to the previous comment in the plural resource docs ☝️

Reply:
Updated in latest commit.